bucket/zookeeper.json

```diff
@@ -0,0 +1,40 @@
+{
+    "homepage": "https://zookeeper.apache.org/",
+    "description": "High-performance coordination service for distributed applications",
+    "version": "3.4.13",
+    "license": "Apache-2.0",
+    "url": "https://www.apache.org/dist/zookeeper/zookeeper-3.4.13/zookeeper-3.4.13.tar.gz",
+    "hash": "sha1:a989b527f3f990d471e6d47ee410e57d8be7620b",
+    "extract_dir": "zookeeper-3.4.13",
+    "bin": [
+        "bin\\zkServer.cmd",
+        "bin\\zkCli.cmd"
+    ],
+    "persist": [
+        "data"
+    ],
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/oraclejdk8",
+            "java/openjdk"
+        ]
+    },
+    "post_install": [
+        "Copy-Item \"$dir/conf/zoo_sample.cfg\" -Destination \"$dir/conf/zoo.cfg\"",
+        "$dataDirPath = (\"$dir\\data\").replace('\\', '\\\\')",
+        "(Get-Content \"$dir/conf/zoo.cfg\") -Replace '^dataDir=(.+)$', \"dataDir=$dataDirPath\" | Set-Content \"$dir/conf/zoo.cfg\""
```

Comment:
Post install could be simplified and saved some transactions.

1. Remove Copy-Item line
1. `(Get-Content \"$dir\\conf\\zoo_sample.cfg\")... | Set-Content \"$dir\\zoo.cfg\"`